Berksfile

```diff
@@ -3,7 +3,12 @@ source 'https://supermarket.chef.io'
 metadata
 
 group :integration do
-  cookbook 'java'
-  cookbook 'minitest-handler'
-  cookbook 'stash_test', :path => 'test/cookbooks/stash_test'
+  cookbook 'java',
+    # Pending release: ark_java fails when tar is not installed
+    # See: https://github.com/agileorbit-cookbooks/java/issues/263
+    github: 'agileorbit-cookbooks/java'
```

Comment:
We've got some pointers in the Berksfile to unpinned git repos, so I'm wondering whether we should commit the Berksfile lock to eliminate noise from the odd broken commit... Thoughts?

Or we could pin to a specific commit in these repos. Your call